bin/page_traffic_load

```diff
@@ -23,10 +23,11 @@ Slop.parse(help: true) do
   DOC
 
   run do |_opts, args|
-    if args.size == 1
+    if args.empty?
```

Comment:
It'd be good to add something to the commit message explaining this change.

Reply:
have updated commit message to give more details